src/styles/utils/_mixins.scss

```
@@ -0,0 +1,11 @@
+@mixin square($watch-size) {
+  width: $watch-size;
+  height: $watch-size;
+}
+
+@mixin animation {
+  animation-name: watch;
+  transform-origin: bottom;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
```

Додай час до анімації, і назви її більш конкретно
```suggestion
@mixin animation ($time) {
  animation-name: watch;
  transform-origin: bottom;
  animation-iteration-count: infinite;
  animation-timing-function: linear;
  animation-duration: $time;
}
```